docs/docs/config/config.md

```diff
@@ -320,6 +320,14 @@ This technically defaults to true for new users, but false for existing. We
 have set `enter_accept = true` in the default config file. This is likely to
 change to be the default for everyone in a later release.
 
+## vim
+Default: true
```

Comment:
The documentation says that the vim mode is enabled by default, but the actual default setting seems to be `false`.

https://github.com/atuinsh/atuin/pull/1553/files#diff-40a913e7519b21e144292a549971f0c8c3cc3ed149336446de8a4fbed16ce07fR440

Maybe I miss something?

Reply:
You're not, this needs updating. It will also need moving to https://github.com/atuin/docs, as this is all about to be deleted